modules/nf-core/evigene/tr2aacds/meta.yml

```diff
@@ -0,0 +1,51 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "evigene_tr2aacds"
+description: Uses evigene/scripts/prot/tr2aacds.pl to filter a transcript assembly
+keywords:
+  - sort
+  - example
+  - genomics
```

Comment:
Are these the default keywords?

Reply:
Yes, good spotting. I have updated the keywords now. Thanks!